src/pages/ImportData.test.js

```diff
@@ -0,0 +1,27 @@
+import { isProtected } from './ImportData';
+
+describe('isProtected', () => {
+  const protectedUrls = [
+    { url: 'https://prod.anvil.gi.ucsc.edu/file', format: 'pfb' },
+    { url: 'https://anvilproject.org/file2', format: 'PFB' },
+    { url: 'https://dev.anvil.gi.ucsc.edu/manifest/files', format: 'pFb' },
+    { url: 'https://gen3.biodatacatalyst.nhlbi.nih.gov/explorer', format: 'PFB' },
+  ];
+  const nonProtectedUrls = [
+    { url: 'https://nonanvil.site.org/file', format: 'pfb' },
+    { url: 'https://prod.anvil.gi.ucsc.edu/file', format: 'entitiesJson' },
+    { url: 'https://google.com/file.pfb', format: 'PFB' },
+    { url: 'https://nonanvil.site.org/file', format: 'tdrexport' },
+    { url: 'https://prod.anvil.gi.ucsc.edu/file', format: 'snapshot' },
+    { url: 'https://anvilproject.org/', format: 'catalog' },
+    { url: 'https://gen3.biodatacatalyst.nhlbi.nih.gov/explorer', format: 'snapShot' },
+  ];
+
+  test.each(protectedUrls)('%f with filetype %f should not be protected', ({ url, format }) => {
+    expect(isProtected(url, format)).toBe(true);
+  });
+
+  test.each(nonProtectedUrls)('%f with filetype %f should not be protected', ({ url, format }) => {
+    expect(isProtected(url, format)).toBe(false);
+  });
```

Comment:
minor nit on the test titles: use `%o` to format an object, instead of `%f` which formats a float; also the two titles should be different!

first title:
```javascript
test.each(protectedUrls)('%o should be protected', …
```

second title:
```javascript
test.each(nonProtectedUrls)('%o should not be protected',
```

Reply:
Ah yeah I put in %f as a placeholder and forgot to go back and look up proper syntax!  Thank you

Reply:
I don't see any uses of `%o` in the terra-ui code base. It looks like string interpolation is the most common way to write this message in tests, e.g.,
```
test.each(protectedUrls)(`$url with format $format should not be protected`,({ url, format }) => {
```

But maybe we should be using `%o` instead? Since I don't have a lot of experience writing javascript code I tend to follow existing conventions in the terra-ui codebase but that doesn't mean it's the best way.